### DISCUSSION TITLE
Report Missing Images

### DISCUSSION THREAD

**Discussion by lukePeavey** [lukePeavey/quotable#52] [Quotable Data]
Report authors that are missing profile images. Include the name of the author.

**Comment by rudrankriyam** [comment]
I found a few - 
- helmut-schmidt
- neil-gaiman
- amy-poehler
- eric-hoffer
- larry-page

**Reply by rudrankriyam** [reply]
Is there a way I can contribute the missing pictures?

**Reply by lukePeavey** [reply]
There is no way to contribute images unfortunately. Maybe in the future. 

I will work on adding the missing images you reported though

**Reply by lukePeavey** [reply]
Done!

**Reply by rudrankriyam** [reply]
Thank you so much!

**Reply by lukePeavey** [reply]
No problem!

**Comment by lukePeavey** [comment]
Fixed all missing and broken profile images for current authors.

**Comment by HaSS-1** [comment]
Hello Luke I'm trying to get author profile image but unfortunately no way to get it done , 
there is no image links array when  fetch('https://api.quotable.io/random') 
Do you please have any idea ?

**Reply by rudrankriyam** [reply]
I believe the endpoint for it is: https://images.quotable.dev/profile/{size}/{author_slug}.jpg

**Reply by HaSS-1** [reply]
Does this mean i have to change all my code : 
`  const getQuote = () => {
    fetch('https://api.quotable.io/random')
      .then(res => res.json())
      .then(data => {
        setQuote(data.content);
        setAuthor(data.author);
        setAuthorImg(data.authorImg);
        console.log(data);
      })
  };`
  
  and i'm rendring so : 
  `<p id='quote'><FontAwesomeIcon className='quote_icon' icon={faQuoteLeft} />{quote}<FontAwesomeIcon icon={faQuoteRight} /></p>
        <p>- {author}</p>
        <img src={authorImg} alt={author} />`
        
        PS: im learning react js
        Thank you in advance
